crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -242,3 +305,21 @@ fn chain_id_nonce_from_request(request: &Request) -> (Vec<u8>, u32) {
         signed_tx.unsigned_transaction().nonce,
     )
 }
+
+const EXAMPLE_ETH_TX_JSON: &str = r#"{
```

Comment:
I think derek just pulled something off of etherscan

Reply:
Took a random uniswap v2 swap from etherscan and used that instead. The transaction can be looked up by its `hash` field on any block explorer so I dont think there's a need to add an etherscan link here but happy to add it if you think I should.

Reply:
But it is useful because it saves me a step to Google to figure it out. Adding the link helps others have the same context that you have while you write it.